tdrs-backend/.env.example

```diff
@@ -17,6 +17,13 @@ JWT_CERT_TEST=a_public_cert
 # Initial superuser username should be the developer's login.gov email address
 DJANGO_SU_NAME=dev.email@example.com
 
+
+### AMS OpenID vars ###
+
+AMS_CONFIGURATION_ENDPOINT=
+AMS_CLIENT_ID=
+AMS_CLIENT_SECRET=
```

Comment:
These are required to be set with at least dummy values when running locally.

Reply:
per discussion with Jorge--clarified that these need to be set to the correct values locally.